config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Didn't realise FRAE didn't have one of these! Argh. I'll make a JIRA ticket...

Reply:
https://eaflood.atlassian.net/browse/RUBY-1468

Reply:
It must have come from `rails app:update` (as per the new_framework_defaults`)